samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -878,9 +878,6 @@ class SamzaContainer(
    * @throws SamzaException, Thrown when the container has already been stopped or failed
    */
   def shutdown(): Unit = {
-    if (status == SamzaContainerStatus.STOPPED || status == SamzaContainerStatus.FAILED) {
```

Comment:
Rather than removing this if-block, do we think it's useful to add a log-message at warn-log level(rather than throwing up).